### PR TITLE
Changed SSOwat config on install

### PR DESCRIPTION
**Pull request by LukeOssevoort**
## Problem

- App would break when passed the Auth_header from SSOwat per issue #20 

## Solution

- Added lines to the install script to set the auth_header variable to false

## PR Status

- [x] Code finished and ready to be reviewed/tested

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -107,6 +107,14 @@ ynh_script_progression --message="Starting a systemd service..." --weight=3
...
 ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="Listening on"
 
+#=================================================
+# SETUP SSOWAT
+#=================================================
+ynh_script_progression --message="Configuring permissions..." --weight=3
+
+ynh_permission_url --permission="main" --auth_header="false"
+ynh_permission_url --permission="api" --auth_header="false"
+
```
Comment (Thovi98) [inline scripts/install:117]: This was the old way to change permissions. Now all is done in manifest.toml :)
```suggestion
```

**Review by Thovi98 — CHANGES_REQUESTED** [review]

File: scripts/install (hunk 1 above)
Comment (Thovi98) [reply scripts/install:117]: (didn’t find the way to suggest on manifest oops)

**Comment by Thovi98** [comment]
Thank you, I already tried that, but didn’t thought about the api!

**Comment by Thovi98** [comment]
https://github.com/YunoHost-Apps/audiobookshelf_ynh/pull/81

Please see how I did it.
I will try it now.

Don’t hesitate to change this in your PR so it gives you the credits :)

**Comment by Thovi98** [comment]
Just tested, and it works nicely!
Thanks @LukeOssevoort 🎉

**Comment by LukeOssevoort** [comment]
Thanks for the advice. I kind of cobbled together my solution from another app on my instance that did the same thing. I will adjust my PR soon

File: scripts/install (hunk 1 above)
Comment (LukeOssevoort) [reply scripts/install]: Moved the auth_header changes to the manifest. Should be good to go now.

File: scripts/install [hunk 2]
```diff
@@ -107,6 +107,11 @@ ynh_script_progression --message="Starting a systemd service..." --weight=3
 # Start a systemd service
 ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="Listening on"
 
+#=================================================
+# SETUP SSOWAT
+#=================================================
+ynh_script_progression --message="Configuring permissions..." --weight=3
+
```
Comment (Thovi98) [inline scripts/install:114]:
```suggestion
```

**Review by Thovi98 — APPROVED** [review]

**Comment by Thovi98** [comment]
Thank you again!

**Pull request merged**